inc/contract.class.php

```diff
@@ -1269,71 +1267,68 @@ static function cronContract($task = null) {
          // Get contrats with periodicity alerts
          $query_periodicity = "SELECT `glpi_contracts`.*
                                FROM `glpi_contracts`
-                               WHERE `glpi_contracts`.`alert` & ".pow(2, Alert::PERIODICITY)." > '0'
-                                     AND `glpi_contracts`.`entities_id` = '".$entity."' ";
+                               WHERE `glpi_contracts`.`alert` & " . pow(2, Alert::PERIODICITY) . " > '0'
```

Comment:
Could you please convert this one to use the new querying method?

Reply:
In facts, the `&` operator has been added (but is missing in the doc):
```php
$crit = [
   'FROM' => 'glpi_contracts',
   'WHERE' => [
      'alert'        => ['&', pow(2, Alert::PERIODICITY)],
      'entities_id'  => $entity,
      'is_deleted'   => 0
   ]
];
//SELECT * FROM `glpi_contracts` WHERE `alert` & '64' AND `entities_id` = '0' AND `is_deleted` = '0'
```

That said, the `> 0` part cannot be added, but I'm not sure this is required

Reply:
@trasher After some tests, I agre with you, the > 0 part will not have a real role : 0&0 => false  / 0&1 => false / 1&1 => true

I will update the PR. Thanks !
Note : i made a PR for the docdev